docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
-## Deprecation Warnings
+### Incremental Updates
+
+In Katello 2.1 we provided a first look at incremental updates via the API. With the 2.2 release, incremental updates can be applied via the API, UI and CLI. Incremental updates are applying errata, packages or puppet modules to a content view which produces a minor version of that content view version. The updated version can then be pushed out to all the environments that that major version exists in, thereby making small updates available to any hosts that are using that particular major version of the content view. Users can find this functionality by visiting the errata details page and clicking on the 'Content Hosts' tab. 
```

Comment:
We can currently only apply errata through the UI, not sure if you want to mention that or just mention errata for now.

>  by visiting the errata details page

Maybe say by visiting the errata index page?  I wasn't sure what errata details page you meant, my first thought was from within the Content Host pages for some reason.